vector/src/main/java/im/vector/app/features/spaces/explore/SpaceDirectoryController.kt

```diff
@@ -135,11 +137,19 @@ class SpaceDirectoryController @Inject constructor(
                         matrixItem(matrixItem)
                         avatarRenderer(host.avatarRenderer)
                         topic(info.topic)
+                        errorLabel(
+                                error?.let {
+                                    host.stringProvider.getString(R.string.error_failed_to_join_room, host.errorFormatter.toHumanReadable(it))
+                                }
```

Comment:
same remark

Reply:
No I can't, if null is past the error formatter returns "Sorry, an error occurred"

Reply:
My bad